db/migrate/20151129025719_create_settings.rb

```diff
@@ -1,16 +1,14 @@
 class CreateSettings < ActiveRecord::Migration[4.2]
   def self.up
-    unless ActiveRecord::Base.connection.table_exists?('settings')
-      create_table :settings do |t|
-        t.string  :var,        null: false
-        t.text    :value,      null: true
-        t.integer :thing_id,   null: true
-        t.string  :thing_type, null: true, limit: 30
-        t.timestamps
-      end
-
-      add_index :settings, %i(thing_type thing_id var), unique: true
+    return if ActiveRecord::Base.connection.table_exists?('settings')
+    create_table :settings do |t|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 't' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->